libs/s25client/s25client.cpp

```diff
@@ -313,10 +315,13 @@ bool InitDirectories()
             return false;
         }
     }
+    const std::string soundLSTPath = RTTRCONFIG.ExpandPath(s25::folders::lstsUser) + "/SOUND.LST";
```

Comment:
this full file path should be added to the files-section too

Reply:
I'm planning on putting this into a function so it is clear what it does: migration from old folders to new (changed names, no longer used...)
I wouldn't put those into a global file as those are not meant to be used anywhere but here

Reply:
Done in: https://github.com/Return-To-The-Roots/s25client/pull/1254/commits/45104bf8cf7b3f32c506a33387fe4b23aa3c2299